test.sh

```diff
@@ -1,18 +1,19 @@
-#!/usr/local/bin/fish
+#!/usr/bin/env bash
 
 ./build.sh
 
 mkdir -p build
 pushd build
 
 for n in ../*.nic.tar
-	set nicname (string replace -r 'iphone_(.*).nic.tar' '$1' (basename $n))
-	set projname $nicname"_example"
-	nic.pl -p $nicname -n $projname -u X --nic $n
-	pushd (string replace "-" "" $projname)
+do
+	nicname=$(basename $n .nic.tar)
+	projname=${nicname#*_}"_example"
+	echo | $THEOS/bin/nic.pl -p $nicname -n $projname -u X --nic $n
```

Comment:
Why was the `echo |` added?

Reply:
It takes advantage of the newline character echo generates to basically simulate an 'enter' press. 

I added it because upon running nic.pl, a few of the template-specific fields were still unpopulated and the user was prompted. I figured it'd be best to get around this by going with the default values (i.e., by leaving the field blank and pressing enter) automatically.